### PR TITLE
refactor: reorganize commands to emphasize playbook and project

### LINKED ISSUE #23: Moving existing projector functionality into new groupings
# This is a Feature Proposal

## Description

Projector's command structure should reflect #5, this issue sets out to tackle reordering existing work to match that structure.

Similar or dependent issues:

- #5

### PR DESCRIPTION
**Pull request by zmmille2**
## What did you implement:

Closes #23

drop commands that interact with things we want to drop support on
rename subcommands to fit parent naming scheme

## How did you implement it:

Purely file deletes, renames, and variable renames.
No functional changes.

## Todos:

_**Note: Run `npm run check` to run all validation checks on proposed changes**_

- [X] Ensure there are no lint errors
       **Validate via `npm run lint`**  
       _Note: Some reported issues can be automatically fixed by running `npm run lint:fix`_
- [X] Write tests and confirm existing functionality is not broken
       **Validate via `npm test`**
- ~~[ ] Write documentation~~ blocked by #3 
- [X] Pull request title and commit message follows [the conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0-beta.2/#summary)
- [X] Provide verification config / commands / resources
- [X] Requested reviews from at least 2 individuals
- [X] Assigned PR to myself
- ~~[ ] Added PR to correct project~~ No projects yet
- [X] This is ready for review

**_Is it a breaking change?:_** YES

### CODE REVIEW

File: src/commands/project/template/import/projectTemplateImport.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export interface AgileImportOptions extends AgileProviderOptions {
   format: string;
 }
 
-export const agileWorkImportCommandFactory = (): Command => {
+export const projectTemplateImportCommandFactory = (): Command => {
   return new Command<AgileImportOptions>()
     .name("import")
     .description("Work Item Import")
```
Comment (elizabethhalper) [inline src/commands/project/template/import/projectTemplateImport.ts:15]: nit: maybe a description of "Project Template Import"?

**Review by elizabethhalper — APPROVED** [review]
looks great! had one nit...glad it wasn't actually 57 files :)

File: src/commands/project/project.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Command } from "../../extensions";
+import { projectSprintsCommandFactory, projectTemplateCommandFactory } from ".";
+
+export const projectCommandFactory = (): Command => {
+  return new Command()
+    .name("playbook")
```
Comment (alancast) [inline src/commands/project/project.ts:6]: Is this supposed to be playbook or something else?

**Review by alancast — APPROVED** [review]
Would be nice to update some documentation as part of this, but _i think_ if I remember correctly that is autogenerated so that will happen on it's own correct? If that doesn't happen on it's own please update some docs with what we removed/renamed after this

**Pull request merged**